samples/StandaloneApp/Pages/Counter.cshtml

```diff
@@ -1,14 +1,15 @@
 ﻿@page "/counter"
+@using Microsoft.AspNetCore.Blazor
 <h1>Counter</h1>
 
 <p>Current count: @currentCount</p>
 
-<button @onclick(IncrementCount)>Click me</button>
+<button onclick="@IncrementCount">Click me</button>
 
 @functions {
     int currentCount = 0;
 
-    void IncrementCount()
+    void IncrementCount(UIMouseEventArgs e)
```

Comment:
I didn't mess with it, because I wanted to get some feedback first, but I had two main thoughts while doing this porting that might improve usability....

1. We could add a special case for `Action` (no arg) for the case where you don't need the args, if we think it will be common. This would be pretty easy for us to do, but we would need to push knowledge of this case through a few levels of the stack.

2. We might want to consider adding `@using Microsoft.AspNetCore.Blazor` to the default usings. Now that we've got more useful stuff you'll want, we might want to do this as a practical matter. This will help keep docs and articles concise.

Reply:
I'm going to go ahead with this PR as is, then I'll follow up on these two items separately. The `Action` one carries some risk of not making it.